docs/content/developers/design-keuzes.md

```diff
@@ -293,3 +293,77 @@ en [Facebook](https://developers.facebook.com/docs/graph-api/reference/photo/#Cr
 Verschillende bronnen zijn hier wel over verdeeld, zoals
 [REST API tutorial](https://restfulapi.net/resource-naming/) en [Wikipedia](https://en.wikipedia.org/wiki/Representational_state_transfer#Relationship_between_URL_and_HTTP_methods)
 maar er is gekozen om te kijken naar de praktijk en DSO.
+
+## Zaak afsluiten
+
+Een zaak wordt afgesloten door een eindstatus toe te kennen aan een `ZAAK`.
+Elk `ZAAKTYPE` moet (minimaal) 1 eindstatus kennen. De eindstatus binnen een
+`ZAAKTYPE` wordt afgeleid uit de `ZAAKSTATUS` met het hoogste `volgnummer`.
+
+Het toekennen van deze `STATUS` bepaald ook een logisch af te leiden
+`ZAAK.einddatum`; dit is namelijk de datum en tijd waarop de eindstatus is
+toegekend. `ZAAK.einddatum` is daarom ook een alleen-lezen attribuut van een `ZAAK`.
+
+Als een ZAAK een eindstatus heeft, betekent dat op het moment van schrijven
+niet dat een ZAAK niet meer aangepast mag worden. Dit wordt voorlopig een
```

Comment:
Moet m.i. zijn: "Als een ZAAK een eindstatus heeft dan is de zaak afgesloten en mogen gegevens van de zaak niet meer aangepast worden (behalve om redenen van correctie). Dit is voorlopig een ..."